packages/manager/modules/web-paas/src/components/additional-option/constants.js

```diff
@@ -1,9 +1,9 @@
 export const ADDON_FAMILY_STAGING_ENVIRONMENT = 'staging_environment';
 export const STORAGE_MULTIPLE = 5;
-export const DEFAULT_ENVIRONMENT = 2;
+export const DEFAULT_ENVIRONMENT_COUNT = 2;
```

Comment:
Can we reuse this from packages/manager/modules/web-paas/src/web-paas.constants.js?